homeassistant/components/group/reproduce_state.py

```diff
@@ -0,0 +1,24 @@
+"""Module that groups code required to handle state restore for component."""
+from typing import Iterable, Optional
+
+from homeassistant.core import Context, State
+from homeassistant.helpers.typing import HomeAssistantType
+from homeassistant.loader import bind_hass
+
+
+@bind_hass
+async def async_reproduce_states(hass: HomeAssistantType,
+                                 states: Iterable[State],
+                                 context: Optional[Context] = None) -> None:
+    """Reproduce component states."""
+    from . import get_entity_ids
+    from homeassistant.helpers.state import async_reproduce_state
```

Comment:
This can be moved to the top of the module.

Reply:
Yeah, just this line.

Reply:
Nope. Not until switch has been converted to new system.

```
tests/helpers/test_state.py:12: in <module>
    from homeassistant.helpers import state
homeassistant/helpers/state.py:17: in <module>
    from homeassistant.components.switch.mysensors import (
homeassistant/components/switch/__init__.py:20: in <module>
    from homeassistant.components import group
homeassistant/components/group/__init__.py:26: in <module>
    from .reproduce_state import async_reproduce_states  # noqa
homeassistant/components/group/reproduce_state.py:5: in <module>
    from homeassistant.helpers.state import async_reproduce_state
E   ImportError: cannot import name 'async_reproduce_state'

```

Reply:
Ok. :+1: